addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -5379,6 +5381,48 @@ public Action Command_PRinfo(int client, int args)
 		default:{
 			CPrintToChat(client, "DEFAULT");
 			db_selectPRinfoUnknownWithMap(client, g_MapRank[client], g_szMapName, 0, g_szSteamID[client]);
+			}
+		}
+
+	return Plugin_Handled;
+}
+
+public Action Command_CCP(int client, int args)
+{
+
+	if (!IsValidClient(client))
+		return Plugin_Handled;
+
+	//CCP MENU
+	ccp_menu = new Menu(CCPMenuHandler);
```

Comment:
Why you're using ccp_menu as a global variable here?

Reply:
did this quite a lkong time ago xD, but after re-reading, i think i had to use this because the way i did the sql portion of the code, there is a point where i have to retrieve the stage rank individually for each stage with a individual query, so if i would create the menu in the final `displayccpmenu()` it would only show 1 stage since all the others stages would be overwritted by the final stage query fetch, hope this isnt confusing xD